_ont/ont-genexis-ft-g2110c.md

```diff
@@ -65,7 +70,113 @@ This ONT supports dual boot.
+
+```sh
+# nv setenv GponSn ZTEGaaa01234
+```
+Reboot ONT to apply the change
+
+You can check if the serial number was correclty changed using the following command:
+
+```sh
+# diag gpon get serialnumber
+ZTEGaaa01234
+```
+
+## Changing the Hardware Version
+
+```sh
+# nv setenv HV G2110CE2V1D0
+```
+Reboot ONT to apply the change
+
+## Changing the Equipment ID
+
+```sh
+# nv setenv ProdName FT-G2110C-2.5G
+```
+Reboot ONT to apply the change
+
+## Changing the IP address
+```sh
+# /etc/scripts/flash get LAN_IP_ADDR
+LAN_IP_ADDR=192.168.1.1
+# /etc/scripts/flash set LAN_IP_ADDR 192.168.1.1
+ ```
+
+## Getting/Setting the ONT's PLOAM password
+
+{% include alert.html content="The PLOAM password is stored in ASCII format" alert="Info" icon="svg-info" color="blue" %}
+
+```sh
+# /etc/scripts/flash get GPON_PLOAM_PASSWD
+GPON_PLOAM_PASSWD=AAAAAAAAAA
+# /etc/scripts/flash set GPON_PLOAM_PASSWD AAAAAAAAAA
+```
+Looks like Genexis has also included an U-Boot variable to store the PLOAM. On currently known firmwares (5.6.1 and 5.7.0) it is not read by the `omci_app` daemon.
+If you want to be future proof, put your PLOAM also on U-Boot env using the following command on the OS shell:
+
+```sh
+# nv setenv ploampwd 1234567890
+ploampwd=1234567890
+```
+
+## Spoofing firmware version
+
+{% include alert.html content="To change this information you need to upload a modified firmware. Do it at your own risk!" alert="Warning" icon="svg-warning" color="red" %}
+
+- Download the modified firmware from [here](https://mega.nz/file/yspFRTJT#JjeDS7NwLZxCzFklQGMJVIuPawo8utQRGbMGEXQv84M) md5:  `91946ad8aada2e469b7fff1d38ee7474`
+- Logon on OS shell of the ONT and run this command: `iptables -F`
+- Open a web browser and go to `http://192.168.1.1/upgrade.asp` or `http://192.168.100.1/upgrade.asp`, enter same credentials used for telnet session, select the modified firmware and click on `Upgrade`
+- After upgrade is done and you are able to reach the ONT via telnet, logon back and run these commands to change firmware version on both banks:
+
+```sh
+# /etc/scripts/flash set OMCI_OLT_MODE 3
+OMCI_OLT_MODE=3
+# nv setenv sw_custom_version0=V6.0.10N14
+# nv setenv sw_custom_version1=V6.0.10N14
+```
+
+Reboot the ONT and check if version was correct applied with this command (take a look at `Version` field):
+
+```sh
+# omcicli mib get 7
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+SWImage
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+=================================
+EntityID: 0x00
+Active: 0
+Committed: 0
+Valid: 1
+*Version: V6.0.10N14*
+ProductCode:
+ImageHash: 0x00000000000000000000000000000000
+=================================
+=================================
+EntityID: 0x01
+Active: 1
+Committed: 1
+Valid: 1
+*Version: V6.0.10N14*
+ProductCode:
+ImageHash: 0x00000000000000000000000000000000
+=================================
+```
 
 # Known Bugs
+- Modified image (based on C-5.7.1-DEV3) has Web Gui enabled, but without attached fiber the daemon will crash
```

Comment:
in what sense does it crash?

Reply:
If you call the web page without fiber attached, you will get the first page and the a server error and the boa daemon dies, so only way to restore http server is to start manually or reboot the whole ONT

Reply:
OK maybe it should be written better